src/bloqade/squin/wire.py

```diff
@@ -50,6 +50,14 @@ class Unwrap(ir.Statement):
     result: ir.ResultValue = info.result(WireType)
 
 
+@statement(dialect=dialect)
+class Wired(ir.Statement):
+    traits = frozenset()
+
+    qubits: tuple[ir.SSAValue, ...] = info.argument(QubitType)
+    body: ir.Region = info.region(multi=False)
```

Comment:
I think this needs to enable `multi` otherwise you lost the ability to have a complete CFG - which is completely legit.